### PR TITLE
List clusters template revamp

### PR DESCRIPTION
**Pull request by fabriziosestito**
This PR updates the cluster list view with the filters, pagination and health component.
Added autocomplete behaviour to some of the the filter inputs, such as `name` and `sid`

Structs were added to abstract the clusters list.

### CODE REVIEW

File: web/clusters.go [hunk 1]
```diff
@@ -261,16 +263,183 @@ func (nodes Nodes) PassingCount() int {
...
+		var sids []string
+
+		hl, ok := hosts[id]
+
+		if ok && len(hl) > 0 {
+			health = hl.Health()
+			set := make(map[string]struct{})
+
+			for _, h := range hl {
+				ss, err := h.GetSAPSystems()
```
Comment (arbulu89) [inline web/clusters.go:292]: I don't really think we should do this `GetSAPSystems`. We should get the SID from the cluster resources (it should be in some cluster node attribute).
In fact, I'm not sure if the cluster should have multiple sids. It's true that the cost-optimized manages 2 sids though

File: web/templates/blocks/health_icon.html.tmpl [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{{ define "health_icon" }}
```
Comment (arbulu89) [inline web/templates/blocks/health_icon.html.tmpl:1]: I guess we should reuse this template in the hosts list too

File: web/templates/clusters.html.tmpl [hunk 3]
```diff
@@ -1,33 +1,61 @@
...
+            $(document).ready(function () {
+                {{- range $Key, $Value := .AppliedFilters }}
+                $("#{{ $Key }}").selectpicker("val", {{ $Value }});
+                {{- end }}
+                $('#clean').click(function () {
+                    $('.selectpicker').selectpicker("deselectAll")
+                });
+            });
+        </script>
+        <select name="health" id="health" class="selectpicker" multiple
```
Comment (arbulu89) [inline web/templates/clusters.html.tmpl:30]: This piece is duplicated too

**Review by arbulu89 — COMMENTED** [review]
@fabriziosestito Nice work!
I just found that there will be some duplicated code between the `hosts` view and the `clusters` view.
Besides that, I did a comment about the usage of this `GetSAPSystems`, which I'm not sure we should use.

Anyway, let's merge and maybe open follow up tickets

File: web/clusters.go (hunk 1 above)
Comment (stefanotorresi) [reply web/clusters.go:292]: to follow up in #143

**Pull request merged**